selfdrive/ui/translations/main_ko.ts

```diff
@@ -479,7 +479,7 @@ location set</source>
         <location filename="../qt/offroad/networking.cc" line="87"/>
         <location filename="../qt/offroad/networking.cc" line="97"/>
         <source>for &quot;%1&quot;</source>
-        <translation>하기위한 &quot;%1&quot;</translation>
+        <translation>&quot;%1&quot;에 접속하려면 인증이 필요합니다</translation>
```

Comment:
The new translation adds more text than "for"? Password is already translated right above

Reply:
The additional translation is better because it is about accessing the wifi ssid